src/lib/app/mainmenu.cpp

```diff
@@ -621,6 +627,10 @@ void MainMenu::init()
     ADD_ACTION("Tools/RssReader", m_menuTools, QIcon(), tr("RSS &Reader"), SLOT(showRssManager()), "");
     ADD_ACTION("Tools/WebInspector", m_menuTools, QIcon(), tr("Web In&spector"), SLOT(toggleWebInspector()), "Ctrl+Shift+I");
     ADD_ACTION("Tools/ClearRecentHistory", m_menuTools, QIcon::fromTheme(QSL("edit-clear")), tr("Clear Recent &History"), SLOT(showClearRecentHistoryDialog()), "Ctrl+Shift+Del");
+
+    m_submenuExtensions = new QMenu(tr("&Extensions"));
```

Comment:
Ah right, this is init(), i got confused with one line above (aboutToHideToolsMenu) :D